### PR TITLE
feat(ramli-component): add ramli-component

### PR DESCRIPTION
**Pull request by mramli24**

### CODE REVIEW

File: src/components/general/atomic/atomic.component.js [hunk 1]
```diff
@@ -22,6 +22,9 @@ const loading = () =>
 const radioButton = () =>
   import(/* webpackChunkName: "radio-button" */ './radio-button/radio-button.component.vue')
 
+const ramlicomp = () =>
+  import(/* webpackChunkName: "ramlicomp" */ './ramlicomp/ramlicomp.component.vue')
```
Comment (irfanandriansyah1997) [inline src/components/general/atomic/atomic.component.js]: @mramli24  saya kira webpackChunkname jadi `general-component`

Comment (irfanandriansyah1997) [inline src/components/general/atomic/atomic.component.js]:
```suggestion
  import(/* webpackChunkName: "general-component" */ './ramlicomp/ramlicomp.component.vue')
```

File: src/components/general/atomic/ramlicomp/ramlicomp.component.vue [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<template>
+  <h1>Ramli Component test</h1>
```
Comment (irfanandriansyah1997) [inline src/components/general/atomic/ramlicomp/ramlicomp.component.vue]: text test nya dihapus aja :dancer:

File: src/components/general/atomic/atomic.component.js (hunk 1 above)
Comment (mramli24) [reply src/components/general/atomic/atomic.component.js]: okay, i will change it soon

**Pull request merged**